code/modules/admin/fun_verbs.dm

```diff
@@ -1119,13 +1126,13 @@
 		else
 			return
 
-	if(!istype(X) || X.gc_destroyed || !SSticker || X.hivenumber != hivenumber_status)
+	if(!istype(X) || X.hivenumber != hivenumber_status)
```

Comment:
Same here, feedback.

Reply:
```suggestion
	if(!istype(X))
		to_chat(usr,"<span class='warning'>Selected xeno is no longer valid.</span>")
		return
	if(X.hivenumber != hivenumber_status)
		to_chat(usr,"<span class='warning'>The hivenumber has changed since we started. Aborting.</span>")
```